metadata/src/main/java/org/apache/kafka/controller/ReplicationControlManager.java

```diff
@@ -956,8 +978,10 @@ void deleteTopic(ControllerRequestContext context, Uuid id, List<ApiMessageAndVe
             throw new UnknownTopicIdException(UNKNOWN_TOPIC_ID.message());
         }
         int numPartitions = topic.parts.size();
+        log.trace("Deleting topic {} with ID {} and {} partitions", topic.name, id, numPartitions);
         try {
             context.applyPartitionChangeQuota(numPartitions); // check controller mutation quota
+            log.trace("Applied partition change quota for topic {} with ID {}", topic.name, id);
```

Comment:
Do we know if there's even a quota set here? There might not be. A better message might be "checked for a partition change quota on topic {} with ID {}"